.github/workflows/publiccode-yml-validation.yml

```
@@ -0,0 +1,12 @@
+name: publiccode.yml validation
+
+on: [pull_request, push]
+
+jobs:
+  publiccode_yml_validation:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
```

```suggestion
      - uses: actions/checkout@v5
```
Please use latest version where possible.